src/include/platform/internal/GenericConfigurationManagerImpl.ipp

```diff
@@ -635,9 +635,13 @@ bool GenericConfigurationManagerImpl<ConfigClass>::IsCommissionableDeviceNameEna
 template <class ConfigClass>
 CHIP_ERROR GenericConfigurationManagerImpl<ConfigClass>::GetCommissionableDeviceName(char * buf, size_t bufSize)
 {
-    ReturnErrorCodeIf(bufSize < sizeof(CHIP_DEVICE_CONFIG_DEVICE_NAME), CHIP_ERROR_BUFFER_TOO_SMALL);
-    strcpy(buf, CHIP_DEVICE_CONFIG_DEVICE_NAME);
-    return CHIP_NO_ERROR;
+    CHIP_ERROR err;
+    
+    err = GetDeviceInstanceInfoProvider()->GetDeviceName(buf, bufSize);
+    SuccessOrExit(err);
+
+exit:
+    return err;
```

Comment:
Can be replaced with just the following?

```c++
return GetDeviceInstanceInfoProvider()->GetDeviceName(buf, bufSize);
```

Reply:
Updated